File: frontend/web_ccadmin_ecommerce/src/app/enterprise/sale/model/dto/SalePaymentDeliveryRegisterDto.ts
import { TrxPaymentEntity } from '../entity/TrxPaymentEntity';
import { TrxPaymentDocumentEntity } from '../entity/TrxPaymentDocumentEntity';

export class SalePaymentDeliveryRegisterDto {
  public OrderToken: string = '';
  public TrxPayment: TrxPaymentEntity = new TrxPaymentEntity();
  public DocumentList: TrxPaymentDocumentEntity[] = [];
}

File: backend/ws_wa_store_ccadmin/src/main/java/com/ccadmin/app/delivery/model/dto/SalePaymentDeliveryRegisterDto.java
package com.ccadmin.app.delivery.model.dto;

import com.ccadmin.app.payment.model.entity.TrxPaymentEntity;
import com.ccadmin.app.payment.model.entity.TrxPaymentDocumentEntity;

import java.util.ArrayList;
import java.util.List;

public class SalePaymentDeliveryRegisterDto {

    public String OrderToken;
    public TrxPaymentEntity TrxPayment;
    public List<TrxPaymentDocumentEntity> DocumentList = new ArrayList<>();
}
